src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -111,28 +131,27 @@ public interface KoverProjectExtension {
     }
 
     @Deprecated(
-        message = "XML report setting was moved to '$REGULAR_REPORT_EXTENSION_NAME { xml { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
+        message = "XML report setting was moved to '$REPORT_EXTENSION_NAME { xml { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
```

Comment:
REPORT_EXTENSION_NAME is `koverReport`, but weren't they moved to `koverReport { defaults { xml {...}}}`?